Package.swift

```diff
@@ -21,9 +21,9 @@ let package = Package(
     ],
     dependencies: [
        .package(
```

Comment:
We should not push this modification. Maybe we could have a hock avoiding this kind of problem..?